.drone.yml

```diff
@@ -176,7 +176,6 @@ pipeline:
   report-coverage:
     image: plugins/coverage
     server: https://coverage.vmware.run
-    token: ${GITHUB_AUTOMATION_API_KEY}
```

Comment:
I pushed the corresponding secret to drone, will push to our internal repo once this PR is in